docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,11 +11,13 @@ tags: ["release-notes"]
 
 <ReleaseNotesVersions />
 
-## Aug 17, 2024 - Release 4.4.X
+## Aug 17, 2024 - Release 4.4.14
 
 <!-- Replace heading ID with the release version below -->
 
-### Palette {#release-4-4-REPLACE_ME}
+This release is specific to Palette Enterprise and does not apply to Palette VerteX.
```

Comment:
Does this mean Self-hosted Palette? I've not see Enterprise many places in the docs.

Reply:
This just means Palette. Jeremy is the one that specifically asked me to use "Palette Enterprise" instead of just Palette lol.

Reply:
@lenny use Palette. We don't use the term Palette enterprise anywhere in our docs